docs/ingestion/appstore.md

```diff
@@ -0,0 +1,60 @@
+# Apple AppStore
+The [AppStore](https://appstore.com/) is an app marketplace developed and maintained by Apple, for mobile apps on its iOS and iPadOS operating systems. The store allows users to browse and download approved apps developed within Apple's iOS SDK. Apps can be downloaded on the iPhone, iPod Touch, or iPad, and some can be transferred to the Apple Watch smartwatch or 4th-generation or newer Apple TVs as extensions of iPhone apps.
+
+Bruin supports AppStore as a source for [Ingestr assets](/assets/ingestr), and you can use it to ingest analytics, sales and performance data into your data warehouse.
+
+In order to set up AppStore connection, you need to add a configuration item in the `.bruin.yml` file and in `asset` file. 
+
+Follow the steps below to correctly set up AppStore as a data source and run ingestion:
+
+### Step 1: Add a connection to .bruin.yml file
+
+To connect to AppStore, you need to add a configuration item to the connections section of the `.bruin.yml` file. This configuration must comply with the following schema:
+
+```yaml
+    connections:
+      appstore:
+        - name: "my-appstore"
+          key_id: "my-key-id"
+          issuer_id: "my-issuer-id"
+          key_path: /path/to/private/key
+
+          # optionally, you can specify the key as a 
+          # base64 encoded string
+          key_base64: "<base64 encoded private key>"
```

Comment:
could we accept the key as is here, and do the base64 encoding ourselves? that would be a lot easier for the users.

Reply:
done